src/pagination/pagination.styles.tsx

```diff
@@ -0,0 +1,213 @@
+
+import { Chevron2LeftIcon } from "@lifesg/react-icons/chevron-2-left";
+import { Chevron2RightIcon } from "@lifesg/react-icons/chevron-2-right";
+import { ChevronLeftIcon } from "@lifesg/react-icons/chevron-left";
+import { ChevronRightIcon } from "@lifesg/react-icons/chevron-right";
+import { EllipsisHorizontalIcon } from "@lifesg/react-icons/ellipsis-horizontal";
+
+import { Color } from "../color";
+import { MediaQuery } from "../media";
+import { Input } from "../input/input";
+import { Text, TextStyleHelper } from "../text";
+
+// =============================================================================
+// STYLE INTERFACE, transient props are denoted with $
+// See more https://styled-components.com/docs/api#transient-props
+// =============================================================================
+interface StyleProps {
+    $selected: boolean;
+}
+interface IconStyleProps {
+    $disabled: boolean;
+}
+
+export const PaginationWrapper = styled.div`
+    display: flex;
+`;
+
+export const PaginationList = styled.div`
+    display: flex;
+    align-items: center;
+    align-self: flex-start;
+    overflow: auto;
+
+    ${MediaQuery.MaxWidth.tablet} {
+        align-self: center;
+    }
+`;
+
+export const PaginationMenu = styled.div`
+    display: inline-flex;
+    align-items: center;
+`;
+
+export const Link = styled(Text.Hyperlink.Small)<IconStyleProps>`
+    color: ${Color.Neutral[1]};
+
+    display: flex;
+    min-width: 2rem;
+    height: 2rem;
+    justify-content: center;
+    align-items: center;
+    cursor: pointer;
+
+    &:hover {
+        color: ${(props) =>
+            props.$disabled ? "black" : Color.Neutral[1]} !important;
+        cursor: ${(props) => (props.$disabled ? "not-allowed" : "pointer")};
+    }
+`;
+
+export const LinkIcon = styled(Text.Hyperlink.Small)<IconStyleProps>`
+    display: flex;
+    min-width: 2rem;
+    height: 2rem;
+    margin: 0 0.5rem;
+    justify-content: center;
+    align-items: center;
+    cursor: pointer;
+`;
+
+export const PaginationJumper = styled.div`
+    display: flex;
+    justify-content: center;
+    margin: 0.625rem;
+    align-items: center;
+    color: ${Color.Neutral[1]};
+
+    display: flex;
+    justify-content: center;
+`;
+
+export const Label = styled(Text.Body)`
+    white-space: nowrap;
+`;
+
+export const InputView = styled(Input)`
+    ${TextStyleHelper.getTextStyle("Body", 400)};
+
+    margin: 0 0.5rem;
+    width: 3.125rem;
+    height: 2.5rem;
+    border-radius: 0.25rem;
+    border: 1px solid ${Color.Neutral[5]};
+    padding: 0.5rem 0.5rem;
+    text-align: center;
+`;
+
+export const Item = styled(Text.Hyperlink.Default)<StyleProps>`
+    ${(props) => {
+        if (props.$selected) {
+            return css`
+                ${TextStyleHelper.getTextStyle("Body", 700)};
+            `;
+        } else {
+            return css`
+                ${TextStyleHelper.getTextStyle("Body", 400)};
+            `;
+        }
+    }}
```

Comment:
can simply do `<Item weight={selected ? "bold" : "regular"} />` to change the font-weight